projects/plugins/jetpack/modules/stats.php

```diff
@@ -1578,11 +1578,11 @@ function stats_get_csv( $table, $args = null ) {
 		'limit'     => 3,
 		'post_id'   => false,
 		'summarize' => '',
+		'blog_id'   => Jetpack_Options::get_option( 'id' ),
```

Comment:
We should provide a PHPDoc type hint for the `blog_id` argument in the Doc block.

Reply:
That's a good point. Done in 43d3c7c9de3cf9a10138e12effec4a8b9e58050f